website/content/api-docs/secret/databases/redis.mdx

```diff
@@ -0,0 +1,80 @@
+---
+layout: api
+page_title: Redis - Database - Secrets Engines - HTTP API
+description: >-
+  The Redis plugin for Vault's database secrets engine generates database
+  credentials to access Redis servers.
+---
+
+# Redis Database Plugin HTTP API
+
+The Redis database plugin is one of the supported plugins for the database
+secrets engine. This plugin generates database credentials dynamically based on
+configured roles for the Redis database.
```

Comment:
`, and also supports [Static Roles](https://developer.hashicorp.com/vault/docs/secrets/databases#static-roles).`

Should we clarify static roles are also supported like it's documented e.g. for MySQL?

Reply:
👍 Will update